### DISCUSSION TITLE
Issu In updating software in CMD

### DISCUSSION THREAD

**Discussion by hsyedmehrab** [microsoft/winget-cli#4071] [General] [answered]
When I tried to update all the software in my windows 10 pc, I got an error message saying that I have to accept all the licences agreements to update all the softswares in my Pc

**Comment by denelon** [comment, marked as answer]
If you have the "msstore" source configured, there is a required source agreement, and for some packages, there is a "package" level agreement that also needs to be accepted to continue.
